.rubocop.yml

```diff
@@ -0,0 +1,1157 @@
+AllCops:
```

Comment:
we usually avoid committing these files, and prefer to load them via [codeclimate prepare](https://codeclimate.com/changelog/582495c32c33066f1b00191d) and then gitignore the file